rules/0575-windows_rules.xml

```diff
@@ -0,0 +1,5170 @@
+  </rule>
+
+  <rule id="20432" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^3089$</field>
+    <description>Cannot save gatherer files</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20433" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^3090$</field>
+    <description>Cannot restore gatherer files from the previous checkpoint</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20434" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^3091$</field>
+    <description>Cannot read checkpoint record</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20435" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^3099$</field>
+    <description>Unable to terminate notifications normally</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20436" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^7011$</field>
+    <description>Directory location invalid</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20437" level="4">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^7013$</field>
+    <description>Disk full, update paused</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20438" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^7040$</field>
+    <description>Index data cannot be read</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20439" level="4">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^7066$</field>
+    <description>Missing directory configuration</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20440" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^7068$</field>
+    <description>Cannot read registry, registry keys missing</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20441" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^1003$</field>
+    <description>The Windows Search Service started</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20442" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^1013$</field>
+    <description>The Windows Search Service stopped normally</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20443" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^7064$</field>
+    <description>Performance monitoring cannot be initialized because the counters are not loaded or the shared memory object cannot be opened</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20444" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^1010$</field>
+    <description>The
```

Comment:
These rules should be included in the groups `windows` and `windows_defender` as they were in the other file.